CMake Error at CMakeLists.txt:348 (find_package):
  Could not find a configuration file for package "libavif" that is
  compatible with requested version "0.9.1".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/libavif/libavif-config.cmake, version: 1.0.0

--- CMakeLists.txt.orig	2023-02-07 00:38:32 UTC
+++ CMakeLists.txt
@@ -81,7 +81,6 @@ cmake_dependent_option(SDL2IMAGE_PNG_SAVE "Add PNG sav
 cmake_dependent_option(SDL2IMAGE_JPG_SAVE "Add JPEG save support" ON SDL2IMAGE_JPG OFF)
 cmake_dependent_option(SDL2IMAGE_PNG_SAVE "Add PNG save support" ON SDL2IMAGE_PNG OFF)
 
-set(LIBAVIF_MINIMUM_VERSION "0.9.1")
 if(SDL2IMAGE_VENDORED AND SDL2IMAGE_AVIF)
     set(SDL2IMAGE_AVIF_VENDORED ON)
 else()
@@ -345,7 +344,7 @@ if(SDL2IMAGE_AVIF)
         endif()
     else()
         message(STATUS "${PROJECT_NAME}: Using system libavif")
-        find_package(libavif ${LIBAVIF_MINIMUM_VERSION} REQUIRED)
+        find_package(libavif REQUIRED)
         list(APPEND PC_REQUIRES libavif)
     endif()
     if(SDL2IMAGE_AVIF_SHARED)
